build.sh

```diff
@@ -259,30 +254,32 @@ bdist() {
     local package name srcdir licensedir zipdir prev_ver_suffix
 
     # Rebuild OpenSlide if suffix changed
-    prev_ver_suffix="$(cat ${build_bits}/.suffix 2>/dev/null ||:)"
+    prev_ver_suffix="$(cat ${os}/${build_bits}/.suffix 2>/dev/null ||:)"
     if [ "${ver_suffix}" != "${prev_ver_suffix}" ] ; then
         clean openslide
-        mkdir -p "${build_bits}"
-        echo "${ver_suffix}" > "${build_bits}/.suffix"
+        mkdir -p "${os}/${build_bits}"
+        echo "${ver_suffix}" > "${os}/${build_bits}/.suffix"
     fi
 
     (
         override_lock
         override_init
+        get_artifacts
         build
         override_remove
     )
 
-    zipdir="openslide-win${build_bits}-${pkgver}"
+    zipdir="openslide-${os}${build_bits}-${pkgver}"
```

Comment:
I think the proper thing to do would be to specify OS and architecture, in case we want to add eg Linux arm64 in future

Reply:
How do you feel about switching from `-m32`, `-m64` to `-mi686` `-mx86_64`, `-marm64`? Or should I add a new `-a` flag

Reply:
Since we're not going to support every combination of OS and CPU, it probably makes sense to have `-m` fully specify the platform, with values like `win-ia32` or `mac-amd64`.  We can then split that out into separate variables internally as needed.

Reply:
I misread this so currently it's now specified as `-o win -m x86_64`; I'll change that now